builds.sh

```diff
@@ -130,11 +147,18 @@ for sourceId; do
 				end
 			')"
 
+			if [ -v "imageArchResolved["$lookup"]" ]; then
```

Comment:
```suggestion
			if [ -n "${imageArchResolved[$lookup]+x}" ]; then
```

It would be clearer/cleaner/more idiomatic to use the above formulation. Plus, while `$lookup` should never have a space in it, it is also safer/future proof.

What I'm not clear on is whether you should use `:+` or `+`. The latter (the one recommended in the suggestion) maps more closely to the result of `-v`. The question is whether it is legitimate for resolved architectures for an image to be null?

Reply:
It looks like it can be empty, so just `+` is probably better.

Reply:
But `:+` if we cache the empty value.

Reply:
Done